.env.example

```
@@ -1,10 +1,10 @@
 DATABASE_URL=postgres://postgres:postgres@localhost:5432/watcher
 SECRET_KEY=dev-secret-key
+ENVIRONMENT=development
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ENVIRONMENT key should go before the SECRET_KEY key

(UnorderedKey)